jsonrpc/server.go

```diff
@@ -387,47 +386,47 @@ func (s *Server) handleWs(w http.ResponseWriter, req *http.Request) {
 			resp, err := s.handler.HandleWs(message, wsConn, req)
 			if err != nil {
 				log.Error(fmt.Sprintf("Unable to handle WS request, %s", err.Error()))
-				_ = wsConn.Load().WriteMessage(msgType, []byte(fmt.Sprintf("WS Handle error: %s", err.Error())))
+				_ = wsConn.WriteMessage(msgType, []byte(fmt.Sprintf("WS Handle error: %s", err.Error())))
 			} else {
-				_ = wsConn.Load().WriteMessage(msgType, resp)
+				_ = wsConn.WriteMessage(msgType, resp)
 			}
 		}
 	}
 }
 
 func (s *Server) increaseHttpConnCounter() {
 	s.connCounterMutex.Lock()
-	atomic.AddInt64(&s.httpConnCounter, 1)
+	s.httpConnCounter++
 	s.logConnCounters()
 	s.connCounterMutex.Unlock()
 }
 
 func (s *Server) decreaseHttpConnCounter() {
 	s.connCounterMutex.Lock()
-	atomic.AddInt64(&s.httpConnCounter, -1)
+	s.httpConnCounter--
 	s.logConnCounters()
 	s.connCounterMutex.Unlock()
 }
 
 func (s *Server) increaseWsConnCounter() {
 	s.connCounterMutex.Lock()
-	atomic.AddInt64(&s.wsConnCounter, 1)
+	s.wsConnCounter++
 	s.logConnCounters()
 	s.connCounterMutex.Unlock()
 }
 
 func (s *Server) decreaseWsConnCounter() {
 	s.connCounterMutex.Lock()
-	atomic.AddInt64(&s.wsConnCounter, -1)
+	s.wsConnCounter--
 	s.logConnCounters()
 	s.connCounterMutex.Unlock()
 }
 
 func (s *Server) logConnCounters() {
-	httpConnCounter := atomic.LoadInt64(&s.httpConnCounter)
-	wsConnCounter := atomic.LoadInt64(&s.wsConnCounter)
+	httpConnCounter := s.httpConnCounter
```

Comment:
You can use directly s.httpConnCounter and s.wsConnCounter, not needed to assign the values to local variables, only to simplify the code